times.go

```diff
@@ -25,28 +25,50 @@ func Time(then time.Time) string {
 	return RelTime(then, time.Now(), "ago", "from now")
 }
 
-var magnitudes = []struct {
+type Seconds int64
```

Comment:
Why is second enough and milliseconds too much?  I work with milliseconds a
lot.

Why isn't second too much?  Minute is enough for humans, right?

On Sun, Aug 23, 2015 at 3:52 AM Victor HU <notifications@github.com> wrote:

> In times.go
> <https://github.com/dustin/go-humanize/pull/26#discussion_r37704778>:
>
> > @@ -25,28 +25,50 @@ func Time(then time.Time) string {
> >  	return RelTime(then, time.Now(), "ago", "from now")
> >  }
> >
> > -var magnitudes = []struct {
> > +type Seconds int64
>
> If @dustin <https://github.com/dustin> don't like new type, I can still
> use unit64 and document the unit.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dustin/go-humanize/pull/26/files#r37704778>.
>